env-config/example.env

```diff
@@ -72,3 +73,11 @@ RUBYONRAILS_LOCAL_PORT=3000 # only needed in local.env
 RUBYONRAILS_RAILS_ENV=
 RUBYONRAILS_SECRET_KEY_BASE=
 
+RUBYONRAILS_APP_DSN=
+RUBYONRAILS_RELEASE_PACKAGE_NAME=
+RUBYONRAILS_APP_ENGINE_SERVICE=application-monitoring-rubyonrails
+RUBYONRAILS_SENTRY_PROJECT=
+RUBYONRAILS_LOCAL_PORT=3000 # only needed in local.env
+SECRET_KEY_BASE=
+RAILS_ENV=
+PIDFILE=
```

Comment:
1. this seems like it already existed in the file right above (lines 68-74)
2. [nit] I'm personally inclined to hardcode PIDFILE in app.yaml.template as implementation detail and not something that we will ever change or affect data in Sentry

Also (mostly curiosity) what's SECRET_KEY_BASE? I can't seem to find where it's used

Reply:
and if you don't mind let's keep all vars prefixed like you had originally for consistency

Reply:
got it 👍 ill make these changes today. thanks for the review @realkosty

Reply:
@realkosty - re `SECRET_KEY_BASE` -> https://cloud.google.com/ruby/rails/appengine#configure_the_rails_secret_key

Reply:
thanks!!